zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -74,22 +76,45 @@ const char * zstdVersion() { return ZSTD_VERSION_STRING; }
 ZEXTERN const char * ZEXPORT z_zlibVersion OF((void)) { return zlibVersion();  }
 
 
+void* ZWRAP_allocFunction(void* opaque, size_t size)
```

Comment:
> Using "static" can be only helpful to remove it from symbols list.

Yes, that's exactly the point.
If it's not intended to be called from outside its own unit, let's `static` that.
In my view, `static` should have been "default", but it's not, so we have to add it manually.